webpack-base.config.js

```diff
@@ -14,7 +14,8 @@ module.exports = new Config().merge({
 
   output: {
     path: path.resolve(__dirname, 'dist'),
-    filename: 'bundle.js'
+    filename: 'bundle.js',
+    publicPath: '/',
```

Comment:
Problemet var ju att man inte kunde navigera sidan med en längre sökväg, t.ex "events/1".
Det visade sig att bundlen laddades i relation till sökvägen och inte till "roten".

Så eftersom att vi använder HtmlWebPackPluginConfig måste publicPath sättas, för att den skall injicera den korrekta adressen till bundlen. Alternativet är att ta bort HtmlWebPackPluginConfig och bara peta in bundlen i en scripttag i index.html. Men, om man sen vill prodsätta appen så kan HtmlWebPackPluginConfig vara användbart då man kanske vill konfigurera vart assets skall servas dynamisk i olika miljöer.